src/plotly/chartDefinition.ts

```diff
@@ -87,48 +88,48 @@ const getChartData = (
         // Get the current X and Y values in the (potentially sparse) series
         const xValue = currentXValues[i];
         const yValue = series.values[i];
-
         // Find which index to update in the (non sparse) totals array.
         const indexToUpdate = uniqueXValues.indexOf(xValue);
-        totals[indexToUpdate] += parseFloat(yValue);
+
+        // yValue over 3 digits will be a string and need to have their ','s removed
+        const formattedYValue = yValue.toString().replace(/,/g, "");
+        totals[indexToUpdate] += parseFloat(formattedYValue);
       }
     }
     let trace: {};
 
     const yHoverInfoPrecision = parseInt(
       chartProps.yAxisProperties.yHoverInfoPrecision as string,
     );
-
     if (isHorizontal) {
       trace = {
         x: series.values,
         y: xValues[seriesIndex].values,
         orientation: "h",
         customdata: totals,
-        hovertemplate: getHoverTemplate(
-          series,
-          "x",
-          "y",
-          yHoverInfoPrecision,
-          isAStackedBar,
-        ),
       };
     } else {
       trace = {
         x: xValues[seriesIndex].values,
         y: series.values,
         orientation: "v",
         customdata: totals,
-        hovertemplate: getHoverTemplate(
-          series,
-          "y",
-          "x",
-          yHoverInfoPrecision,
-          isAStackedBar,
-        ),
       };
     }
 
+    // if interactivity is enabled show hover text over chart
+    if (chartProps.Interactivity.interactivity === "x+y") {
```

Comment:
I've changed it to a const, but I've swapped it over to not equal "none", just in case in the future we wanted to introduce other interactivity modes.
Examples being "x", "y", "x+y", "x+y+z", "all".
https://plotly.com/javascript/reference/scatter/#scatter-hoverinfo